Sources/PointFree/Episode/Show.swift

```diff
@@ -225,11 +226,13 @@ private let videoView = View<(Episode, isEpisodeViewable: Bool)> { episode, isEp
     [
       video(
         [
-          `class`([innerVideoContainerClass]),
+          `class`([innerVideoContainerClass, videoClasses]),
+          style(position(.absolute)),
```

Comment:
the `innerVideoContainerClass` styles contains a `position: absolute`, but it seems video.js overrides that with relative position. so I use this inline style to override that :/